modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -644,7 +646,10 @@ private JSONArray _getFiltersJSONArray(
 					return JSONUtil.put(
 						"active", hasPreloadedData
 					).put(
-						"entityFieldType", FDSEntityFieldTypes.DATE
+						"entityFieldType",
+						Objects.equals(type, "date") ?
+							FDSEntityFieldTypes.DATE :
+								FDSEntityFieldTypes.DATE_TIME
```

Comment:
Forwarding `date-time` fields as new entity type. My initial plan was to simply save `date-time` fields as `date` type  to DSM persistence, but it turns out `date-time` fields require different odata string format, so we need to tell them apart from `date` fields.

Reply:
yes!